<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="fileSelectButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Select file</string>
    </property>
   </widget>
   <widget class="QLabel" name="AfterFileChooseLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose File:</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorLabel">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>821</width>
      <height>371</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>819</width>
       <height>369</height>
      </rect>
     </property>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>801</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Table VIew</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="convertToJson">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>100</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert to JSON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadSort">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Load and sort()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="database">
    <property name="geometry">
     <rect>
      <x>622</x>
      <y>98</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Write to Database</string>
    </property>
   </widget>
   <zorder>scrollArea</zorder>
   <zorder>fileSelectButton</zorder>
   <zorder>AfterFileChooseLabel</zorder>
   <zorder>label</zorder>
   <zorder>errorLabel</zorder>
   <zorder>convertToJson</zorder>
   <zorder>loadSort</zorder>
   <zorder>database</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>858</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
